benchmarks/conftest.py

```diff
@@ -0,0 +1,177 @@
+import os
+from datetime import datetime
+from decimal import Decimal as PyDecimal
+import pytest
+from openpyxl import load_workbook
+from stock_indicators.indicators.common import Quote
+
+dir = os.path.dirname(__file__)
+data_path = os.path.join(dir, "History.xlsx")
```

Comment:
Any chance you can just reuse the `History.xlsx` in the other folder.  Might be less problematic to avoid redundant files.

Reply:
I'll think about better folder structure on next commits.